package.json

```diff
@@ -50,6 +51,8 @@
     "speculate": "^1.7.4"
   },
   "devDependencies": {
+    "@storybook/react": "^4.0.0-alpha.8",
+    "babel-core": "^6.26.3",
```

Comment:
Why is this being added?

Reply:
https://storybook.js.org/basics/guide-react/#add-react-react-dom-and-babel-core